proposals/nnnn-rc-radio-climate-parameter-update.md

```diff
@@ -0,0 +1,106 @@
+
+We propose to add a new parameter `availableHdChannels` to `RadioControlData` to allow an application to know what HD radio channel indexes are available.
+We add a new parameter `climateEnable` to `ClimateControlData` to allow an application to power on or off climate control.
+We also change the minimum index of HD radio sub-channels from 1 to 0.
+
+#### Mobile and HMI API
+
+```xml
+<struct name="RadioControlData" since="4.5">
+...
+<!-- new additions or changes -->
+-   <param name="availableHDs" type="Integer" minvalue="1 maxvalue="7" mandatory="false" since="5.0">
++   <param name="availableHDs" type="Integer" minvalue="0" maxvalue="7" mandatory="false" since="5.1">
+        <description>number of HD sub-channels if available</description>
+        <history>
+            <param name="availableHDs" type="Integer" minvalue="1" maxvalue="3" mandatory="false" since="4.5" until="5.0"/>
++           <param name="availableHDs" type="Integer" minvalue="1" maxvalue="7" mandatory="false" since="5.0" until="5.1"/>
+        </history>
+    </param>
+	
++   <param name="availableHdChannels" type="Integer" minvalue="0" maxvalue="7" mandatory="false" minsize="1" maxsize="8" array="true" since="5.1">
++       <description>the list of hd sub-channel indexes if available, read-only </description>
++   </param>
+	
+-    <param name="hdChannel" type="Integer" minvalue="1" maxvalue="7" mandatory="false" since="5.0">
++    <param name="hdChannel" type="Integer" minvalue="0" maxvalue="7" mandatory="false" since="5.1">
+        <description>Current HD sub-channel if available</description>
+        <history>
+            <param name="hdChannel" type="Integer" minvalue="1" maxvalue="3" mandatory="false" since="4.5" until="5.0"/>
++           <param name="hdChannel" type="Integer" minvalue="1" maxvalue="7" mandatory="false" since="5.0" until="5.1"/>
+        </history>
+    </param>
+...
+</struct>
+
+<struct name="RadioControlCapabilities" since="4.5">
+...
++   <param name="availableHdChannelsAvailable" type="Boolean" mandatory="false" since="5.1">
++       <description>
++           Availability of the list of available HD sub-channel indexes.
++           True: Available, False: Not Available, Not present: Not Available.
++       </description>
++   </param>
+...
+</struct>
+
+
+
+<struct name="ClimateControlCapabilities" since="4.5">
+...
++   <param name="climateEnableAvailable" type="Boolean" mandatory="false" since="5.1">
++       <description>
++           Availability of the control of enable/disable climate control.
++           True: Available, False: Not Available, Not present: Not Available.
++       </description>
++   </param>
+...
+</struct>
+
+<struct name="ClimateControlData" since="4.5">
++    <param name="climateEnable" type="Boolean" mandatory="false" since="5.1">
++     </param>
+...
+</struct>
+```
+
+
+
+
+## Potential downsides
+
+None
+
+## Impacts on existing code
+
+- New parameters need to be updated within the RPC.
```

Comment:
Change to:
- New parameters **will** need to be updated within the RPC.

Reply:
updated